curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/61968e9243a4090cc805531c.md

```diff
@@ -11,10 +11,10 @@ Remove both the horizontal and vertical scrollbars, and prevent programmatic scr
 
```

Comment:
I can't reach it, but the line above this should remove "and prevent programmatic scrolling", as the `clip` property does that but the `hidden` property does not.

Reply:
Line updated @naomi-lgbt